proto/spire/plugin/agent/trustbundlesource/v1/trustbundlesource.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+package spire.plugin.agent.trustbundlesource.v1;
+option go_package = "github.com/spiffe/spire-plugin-sdk/proto/spire/plugin/agent/trustbundlesource/v1;svitrustbundlesourcev1";
+
+import "google/protobuf/timestamp.proto";
+import "spire/plugin/types/bundle.proto";
+    
+service TrustBundleSource {
+    // Get an up to date Trust Bundle
+    rpc GetTrustBundle(GetTrustBundleRequest) returns (GetTrustBundleResponse);
+}
+
+message GetTrustBundleRequest {
+    // The trust domain for which to retrieve the trust bundle
+    string trust_domain = 1;
```

Comment:
Would it ever retrieve bundles for any other trust domain apart from the current one? The current trust domain is already available through https://github.com/spiffe/spire-plugin-sdk/blob/8f411ead94ad0195dc6e896c2c22a91320a0e7a4/proto/spire/service/common/config/v1/config.proto#L63

Reply:
oh, cool. didn't realize that. Lets remove it then.